docs/value/semi-values.md

```diff
@@ -98,6 +98,33 @@ values = compute_banzhaf_semivalues(
 )
 ```
 
+### Banzhaf semi-values with MSR sampling
+Wang et. al. propose a more sample-efficient method for computing Banzhaf semivalues in their paper 
+*Data Banzhaf: A Robust Data Valuation Framework for Machine Learning* [@wang_data_2022].
+This method updates all semivalues per evaluation of the utility (i.e. per model trained) based on whether
+a specific data point was included in the data subset or not. 
+The mathematical formula for computing the semivalues is
+
+$$\hat{\phi}_{MSR}(i) = \frac{1}{|\mathbf{S}_{\ni i}|} \sum_{S \in \mathbf{S}_{\ni i}} U(S)
+        - \frac{1}{|\mathbf{S}_{\not{\ni} i}|} \sum_{S \in \mathbf{S}_{\not{\ni} i}} U(S)$$
+
+where $\mathbf{S}_{\ni i}$ are the subsets that contain the index $i$ and $\mathbf{S}_{\not{\ni} i}$ are the 
+subsets not containing the index $i$.
+
+The function implementing this mehod is
+[compute_msr_banzhaf_semivalues][pydvl.value.semivalues.compute_msr_banzhaf_semivalues].
+```python
+from pydvl.value import *
```

Comment:
I think it's better to import exactly what we need for this snippet:

```python
from pydvl.value import compute_msr_banzhaf_semivalues, RankStability
```

P.S. There should be other snippets that use star imports and we should fix those as well in another PR.